[Docs] be keen to newcomers (#5439)
sonata-project/SonataAdminBundle: docs/cookbook/recipe_image_previews.rst

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

If we are working directly with our ``ImageAdmin`` class then getting hold of

> the ``Image`` instance is a case of calling ``$this->getSubject()``. Since

Reviewer: This sounds weird without "simply"
```suggestion
the ``Image`` instance can be done by calling calling ``$this->getSubject()``. Since
```

Author: Calling twice? 😂

Reviewer: woops :P
